### ISSUE TITLE
Documentations from the standard library not included in the generated HTML docs.

### ISSUE THREAD

**Issue by CCDSY** [realm/jazzy#313]
I got a `struct` that conforms to `SequenceType`, and the documentation for the method `generate()` is provided in the standard library. I can even view it in Xcode.
<img width="502" alt="2015-10-04 7 10 27" src="https://cloud.githubusercontent.com/assets/9509755/10267580/2f7568fc-6acc-11e5-85b0-0a22cf6adfd7.png">
But in the HTML docs generated by jazzy, it says undocumented. Even the declaration is all wrong. 
<img width="429" alt="2015-10-04 7 19 27" src="https://cloud.githubusercontent.com/assets/9509755/10267749/dde2a60a-6ad3-11e5-8350-4ec101d65be5.png">

I believe this is a bug. But if it's not, please tell me how should I set the settings so that (a) it includes standard library docs and (b) makes the declaration display correctly like it does in Xcode.

**Comment by segiddins** [comment]
Can you include some more information about the code you're trying to document so we can reproduce this ?

**Comment by CCDSY** [comment]
The code for the method I mentioned above is just like this. I didn't write any documentation or comment for it, because it's already provided in the standard library for `SequenceType`.
<img width="388" alt="2015-10-04 11 21 14" src="https://cloud.githubusercontent.com/assets/9509755/10268653/b0336028-6af1-11e5-871a-e3f2f516177b.png">

This is the declaration for `generate()` in the generated header file for the standard library. 
<img width="416" alt="2015-10-04 11 23 28" src="https://cloud.githubusercontent.com/assets/9509755/10268654/b1474b96-6af1-11e5-802e-937b49479947.png">

By the way, I didn't set any parameter while I invoke jazzy. I just `cd` to my project directory and said `jazzy`.

**Comment by segiddins** [comment]
Ah. In that case, this is actually a feature request that's being tracked at https://github.com/realm/jazzy/issues/190.

**Comment by CCDSY** [comment]
Then what about the wrong declaration? 
The declaration for `generate()` should be 
```
public func generate() –> Generator
```
or
```
public func generate() –> CircularGenerator<T>
```
not
```
public struct CircularCollection<T> : SequenceType
```

At least, a method declaration should start with a `func` keyword not a `struct`, right?

**Comment by segiddins** [comment]
ah yeah I missed that bit

**Comment by jpsim** [comment]
@CCDSY you seem to have found a bug where if the declaration cannot be parsed, it falls back on the parent declaration.

**Comment by jpsim** [comment]
Fixed in #441.
